pkg/secrets/passphrase/manager.go

```diff
@@ -151,7 +153,35 @@ func NewPassphaseSecretsManagerFromState(state json.RawMessage) (secrets.Manager
 	// This is not ideal, but we don't have a great way to prompt the user in this case, since this may be
 	// called during an update when trying to read stack outputs as part servicing a StackReference request
 	// (since we need to decrypt the deployment)
-	phrase := os.Getenv("PULUMI_CONFIG_PASSPHRASE")
+	phrase := ""
+	passphrase, isOk := os.LookupEnv("PULUMI_CONFIG_PASSPHRASE")
+	if isOk {
+		phrase = passphrase
+	}
+	phraseFile, isOk := os.LookupEnv("PULUMI_CONFIG_PASSPHRASE_FILE")
+	if isOk {
```

Comment:
Is it intentional that this branch runs even if the previous one found a passphrase?  This is slightly unusual structure for the logic here - and feels maybe not like the intended behaviour?